dashboard/app/main.go

```diff
@@ -2074,7 +2075,7 @@ func fetchErrorLogs(c context.Context) ([]byte, error) {
 	var entries []*logging.Entry
 	for len(entries) < maxLines {
 		entry, err := iter.Next()
-		if err == iterator.Done {
+		if errors.Is(err, iterator.Done) {
```

Comment:
I'm not sure it's important, but `iter.Next()` specifically says it returns `iterator.Done` directly.

Reply:
Just to shift responsibility .... It was linter! )
We can try to disable this check type, use nolint or errors.Is().
Last one looks best to me. @dvyukov ?

Reply:
Checked other projects. Everybody uses err == iterator.Done.